src/sentry/static/sentry/app/views/dashboardsV2/widget/new/widgetNew.tsx

```diff
@@ -162,69 +200,91 @@ class WidgetNew extends AsyncView<Props, State> {
             >
               <VisualizationWrapper>
                 <SelectControl
-                  name="visualization"
-                  options={DISPLAY_TYPE_CHOICES.slice()}
-                  value={visualization}
-                  onChange={(option: {label: string; value: Widget['displayType']}) => {
-                    this.handleFieldChange('visualization', option.value);
+                  name="displayType"
+                  options={Object.keys(displayTypes).map(value => ({
+                    label: displayTypes[value],
+                    value,
+                  }))}
+                  value={displayType}
+                  onChange={(option: {label: string; value: DisplayType}) => {
+                    this.handleFieldChange('displayType', option.value);
+                  }}
+                />
+                <WidgetCard
+                  api={this.api}
+                  organization={organization}
+                  selection={selection}
+                  widget={{
+                    title,
```

Comment:
@priscilawebdev We may need an input field to be able to customize the widget title.

Reply:
sure thing, I am created a component called editableText for that :)